airdrop/domain/factory/airdrop_window_factory.py

```diff
@@ -5,4 +5,4 @@ class AirdropWindowFactory:
     @staticmethod
     def convert_airdrop_window_model_to_entity_model(window):
         return AirdropWindow(window.airdrop_id, window.id, window.airdrop_window_name,
-                             window.description, str(window.registration_start_period), str(window.registration_end_period), window.total_airdrop_tokens, window.timelines, str(window.claim_start_period), str(window.claim_end_period), window.airdropwindow_rules).to_dict()
+                             window.description, str(window.registration_start_period), str(window.registration_end_period), window.total_airdrop_tokens, window.timelines, str(window.claim_start_period), str(window.claim_end_period), window.airdropwindow_rules, window.airdrop_window_identifier).to_dict()
```

Comment:
can we keep it as airdrop_window_id instead of airdrop_window_identifier. 
We use airdrop_window_id in other places right? let's stick to the same name everywhere.

Reply:
we already have a field called airdrop_window_id which is the primary key of the airdrop window table